open-api/clients/javascript-client/README.md

```diff
@@ -0,0 +1,179 @@
+# havfen_community_api
+
+HavfenCommunityApi - JavaScript client for havfen_community_api
```

Comment:
Here are some small typos. It should be called Hafven, there is a mix of havfen and hafven in the document. Guess we should call it hafven.